lib/Service/ColumnTypes/TextLinkBusiness.php

```diff
@@ -36,6 +36,8 @@ public function parseValue($value, ?Column $column = null): string {
 		if($data !== null) {
 			if (isset($data['resourceUrl'])) {
 				return json_encode(json_encode([
+					'thumbnailUrl' => $data['thumbnailUrl'],
```

Comment:
You're right the change only affects new links because it seems that since this two lines have been missing every link written to the database lacks of a `thumbnailUrl` (at least in the last versions of tables)... So solving that would maybe need some lookup for links without a `thumbnailUrl` and then generating / searching the correct one (?).